1-org/scc_notification.tf

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  SCC Notification
+*****************************************/
+
+locals {
+  description = "SCC Notification for all active findings"
+}
+
+
+resource "google_pubsub_topic" "scc_notification_topic" {
+  name    = "top-scc-notification"
+  project = module.scc_notifications.project_id
+}
+
+resource "google_pubsub_subscription" "scc_notification_subscription" {
+  name    = "sub-scc-notification"
+  topic   = google_pubsub_topic.scc_notification_topic.name
+  project = module.scc_notifications.project_id
+
+  message_retention_duration = "1200s"
+  retain_acked_messages      = true
+
+  ack_deadline_seconds = 20
+
+  expiration_policy {
+    ttl = "300000.5s"
+  }
+}
+
+
+module "scc_notification" {
+  source  = "terraform-google-modules/gcloud/google"
+  version = "~> 1.1.0"
+
+  platform = "linux"
+
+  additional_components = var.skip_gcloud_download ? [] : ["alpha"]
```

Comment:
I am not sure if we need this check? We would still need to make sure that `alpha` components are installed regardless?

Reply:
Hi @bharathkkb

If we skip download and force the `additional_components = ['alpha']` we got this error:

```
Error: Error running command 'gcloud components install alpha --quiet': exit status 1. Output: ERROR: (gcloud.components.install) 
You cannot perform this action because the Cloud SDK component manager 
is disabled for this installation. You can run the following command 
to achieve the same result for this installation: 

sudo apt-get install google-cloud-sdk
```

So that's why the ternary was added. Any advice?

Reply:
@amandakarina are you running in a system where gcloud was installed using apt/yum? [Docs](https://cloud.google.com/sdk/docs/components#managing_cloud_sdk_components) suggest that may result in component manager being disabled. Could you test it inside the cft container?

Reply:
Hi @bharathkkb running inside the cft container I've got no error.

Reply:
@amandakarina wouldnt it just say? 
```
[root@5602acbb5270 workspace]# gcloud components install alpha
All components are up to date.
```
If you are still running into it, lets keep the ternary. We can revisit later if needed.
I think the check you pointed our is great, will also look into fixing this in the gcloud module.